Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -966,6 +966,9 @@ void function _OnServerSelectedAsync( var button )
 
 	file.lastSelectedServer = serverIndex
 
+	// Count mods that have been successfully downloaded
+	int downloadedMods = 0;
+
 	// check mods
 	for ( int i = 0; i < NSGetServerRequiredModsCount( serverIndex ); i++ )
```

Comment:
some mods might already be downloaded and installed (possibly disabled) so it'd make sense to check first if the mod is available before downloading

Reply:
It seems that enabling a server-required mod that's already present on the client (but disabled) is not done by current code; it's kinda out-of-scope for this PR, and should be done in a separate PR imo.

Reply:
I looked at the code like 8 months ago and it did exactly this, also this shouldn't be a lot of effort to (re)implement

Reply:
Ok so I missed this missing-mod-is-already-locally-there check, it is done after the mods downloading process (and after server authentication too: https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut#L1088).

So it effectively makes sense to not download a mod if it's already installed, I will steal some code from there to ensure not downloading a mod for nothing.

Reply:
Addressed in 34a3b292bd65b5ffea49754ccae904d7fe31a277.